README.md

```diff
@@ -74,6 +74,12 @@ exclude = [
     "./target"
 ]
 
+# Commands to be executed after scaffolding, from within the generated project
```

Comment:
I was thinking about supporting either PRE and POST generation and using something like

```toml
[hooks]
# Commands to be executed before scaffolding, from within the generated project
pre = [
    "cargo vendor",
    "bash -c some_script.sh"
]
# Commands to be executed after scaffolding, from within the generated project
post = [
    "cargo vendor",
    "bash -c some_script.sh"
]
```

What do you think ?

Reply:
That's a good idea, I will implement this